lib/iiif_manifest/v3/manifest_builder/canvas_builder.rb

```diff
@@ -54,6 +56,15 @@ def apply_record_properties
           canvas.label = ManifestBuilder.language_map(record.to_s) if record.to_s.present?
           annotation_page['id'] = "#{path}/annotation_page/#{annotation_page.index}"
           canvas.items = [annotation_page]
+          canvas.thumbnail = [thumbnail] if iiif_endpoint
```

Comment:
Given that we later call `thumbnail.apply(canvas)`, I believe we want to have this line be `apply_thumbnail_to(canvas)`

The `apply` method takes the canvas.  But to understand if this is correct, we need to look at prior art for the various `apply` methods.